iOS/G3MiOSSDK/Commons/Rendererers/PlanetRenderer.cpp

```diff
@@ -32,24 +32,29 @@
 #include "Layer.hpp"
 #include <algorithm>
 
+#include "MaxLevelTileLODTester.hpp"
+#include "ProjectedCornersDistanceTileLODTester.hpp"
+#include "TimedTileLODTester.hpp"
+#include "MaxFrameTimeTileLODTester.hpp"
+
```

Comment:
Why those concrete classes are included here?  It's not enough to include just the super class?  Is there any coupling?